tests/include/ApiTest.php

```diff
@@ -1285,6 +1287,8 @@ public function testApiStatusShow()
 	 */
 	public function testApiStatusShowWithXml()
 	{
+		$this->markTestSkipped('This test requires an item ID');
```

Comment:
There are 6 items at
https://github.com/friendica/friendica/blob/develop/tests/datasets/api.yml

Cannot we take one of them?